YTSpammerPurge.py

```diff
@@ -190,6 +188,21 @@ def main():
   config = files.load_config_file(configVersion)
   os.system(clear_command)
 
+  # Disable colors before they are used anywhere
+  try:
+    if config['colors_enabled'] == False:
+      # Initiates colorama and creates shorthand variables for resetting colors
+      init(autoreset=True, strip=True, convert=False)
+      # Disables colors entirely
+    else:
+      # Initiates colorama and creates shorthand variables for resetting colors
+      init(autoreset=True)
+
+  except Exception as e:
```

Comment:
Maybe you can detect if the option exists instead

Reply:
I'll take any suggestion, as I said, `I'm not 100% sure what exceptions could be raised here.` 😉
It could be a useless `try: except:`, and I could just remove it 👍
I'm not sure 🤷